thriftpy2_httpx_client/aio/transport.py

```diff
@@ -32,6 +32,9 @@ def __init__(self, url: Union[str, httpx.URL], **kwargs):
         self._url = httpx.URL(url)
         self._kwargs = kwargs
         kwargs['base_url'] = self._url
+        self._path = ''
+        if 'path' in kwargs:
+            self._path = kwargs.pop('path')
```

Comment:
Since this isn't being proxied through directly to the HTTPX Client, it should be in an input like so: `__init__(url: str, path: str = '', **kwargs)` and be called out in the docstring.